scripts/add_users.py

```diff
@@ -0,0 +1,33 @@
+import argschema
+import json
+import requests
+
+class RegisterSchema(argschema.ArgSchema):
+    email_list = argschema.fields.List(
+        argschema.fields.String,
+        required=True,
+        cli_as_single_argument=True,
+        description='List of emails to register in the app.'
+    )
+    port = argschema.fields.Int(
+        default=5000,
+        description='Port the app is running on on the localhost.'
+    )
+
+
+class Register(argschema.ArgSchemaParser):
+    """Add a list of users to the app"""
+    default_schema = RegisterSchema
+
+    def run(self):
+        emails = self.args['email_list']
+        for email in emails:
+            requests.post(
```

Comment:
One thing that has to be handled is if this request doesn't work (it returns non 200 response code, which could happen if the email already exists). I think in that case it should be logged

Reply:
Ended up using `print` instead of the logger to print to std out.

Reply:
Ended up using print instead of the logger to print to std out.